keyboards/handwired/split65/stm32/halconf.h

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2020 QMK
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * This file was auto-generated by:
+ *    `qmk chibios-confmigrate -i keyboards/handwired/split65/stm32/halconf.h -r platforms/chibios/common/configs/halconf.h`
+ */
+
+#pragma once
+
+#define HAL_USE_DAC TRUE
+
+#define HAL_USE_GPT TRUE
+
+#define HAL_USE_I2C TRUE
+
+#define HAL_USE_PWM TRUE
+
+#define HAL_USE_SERIAL TRUE
+
+#define HAL_USE_SERIAL_USB TRUE
+
+#define HAL_USE_SPI TRUE
+
+#define PAL_USE_CALLBACKS TRUE
+
+#define PAL_USE_WAIT TRUE
```

Comment:
Only this is needed, everything else is enabled already by using the proton C board config.

```suggestion
#define HAL_USE_SERIAL TRUE
```